test/ffmpeg-vaapi/encode/encoder.py

```diff
@@ -100,6 +100,9 @@ def encode(self):
     if self.mformat is None:
       slash.skip_test("{format} format not supported".format(**vars(self)))
 
+    if self.codec in ["mpeg2"]:
+      self.qp = self.mqp
+
```

Comment:
Please handle the mpeg2 mqp directly in `gen_output_opts` so we don't overwrite self.qp:

How about:
```
    if vars(self).get("qp", None) is not None:
      if self.codec in ["vp8", "vp9",]:
        opts += " -global_quality {qp}"
      elif self.codec in ["mpeg2"]:
        opts += " -global_quality {mqp}"
      else:
        opts += " -qp {qp}"
```

Reply:
have inlined the value into function "gen_output_opts", thanks.